legend-engine-xt-serviceStore-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/from/ServiceStoreParseTreeWalker.java

```diff
@@ -202,7 +261,7 @@ private Service visitService(ServiceStoreParserGrammar.ServiceContext ctx)
 
         // security
         ServiceStoreParserGrammar.SecuritySchemeDefinitionContext supportedAuthCtx = PureGrammarParserUtility.validateAndExtractRequiredField(ctx.securitySchemeDefinition(), "security", this.walkerSourceInformation.getSourceInformation(ctx));
-        service.security = ListIterate.collect(supportedAuthCtx.identifier(), this::visitSecurityScheme);
+        service.securitySchemeRequirements = ListIterate.collect(supportedAuthCtx.identifier(), this::visitSecurity);
```

Comment:
lets make security filed in protocol depreacted if we didn't already

Reply:
we have done that already